openedx/core/djangoapps/user_api/management/tests/test_bulk_user_org_email_optout.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test the test_bulk_user_org_email_optout management command
+"""
+import os
+import tempfile
+import mock
+import pytest
+
+from django.core.management import call_command
+
+pytestmark = pytest.mark.django_db
+
+
+CSV_DATA = """1,UniversityX
+2,CollegeX
+3,StateUX
+"""
+
+
+def _create_test_csv():
+    __, file_name = tempfile.mkstemp(text=True)
```

Comment:
I think this leaves the file around if you don't clean it up. Maybe not a big deal, I'm not sure how the automated testing handles temp dirs, just noting it. Pytest has some temp dir handling that does auto cleanup if you think that it may be a concern: https://docs.pytest.org/en/latest/tmpdir.html or maybe `tempfile.TemporaryFile` instead.

Reply:
I removed the file below in the `os.unlink()` call. But I agree - this should really use a context manager instead to ensure file deletion.